recipes/imas-data-dictionary/conda-forge.yml

```
@@ -0,0 +1,3 @@
+noarch_platforms:
+  - linux_64
+bot: {automerge: true}
```

```suggestion
bot: {automerge: true}
```
The default `noarch_platform` is already `linux_64`, so this is no longer needed.